hw/dv/data/tests/csr_tests.hjson

```diff
@@ -37,6 +37,13 @@
       run_opts: ["+run_same_csr_outstanding"]
       en_run_modes: ["csr_tests_mode"]
     }
+
+    {
+      name: "{name}_csr_mem_rw_with_rand_reset"
+      uvm_test_seq: "{name}_common_vseq"
+      run_opts: ["+run_csr_mem_rw_with_rand_reset"]
+      en_run_modes: ["csr_tests_mode"]
+    }
```

Comment:
Thanks for doing this Weicai, do you mind adding this to `hw/dv/tools/testplan` so we can directly import it into each IP's testplan?

Reply:
Thanks Cindy, I forgot to add it to testplan again. Added to csr_testplan